docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -421,6 +411,12 @@ code: |
   add_draft_intro_screen = True
 ---
 code: |
+  # HACK(brycew): could be a better place to put it, but can't find one :/
```

Comment:
This actually feels fine to me. It's where we're passing it to the generated YAML.